include/CppUTest/Utest.h

```diff
@@ -61,7 +61,12 @@ class Utest
 
 class TestTerminator
 {
+protected:
+    static bool crashOnFail_;
```

Comment:
Class attribute used to hold whether we are running in crash-on-fail mode or not.